lib/auth_web/controllers/auth_controller.ex

```diff
@@ -1,89 +1,235 @@
 defmodule AuthWeb.AuthController do
   use AuthWeb, :controller
-  import Plug.Conn
-  import Comeonin.Bcrypt, only: [checkpw: 2]
-  plug(Ueberauth)
-  # plug :authenticate_user when action in [:index, :show]
-  alias Auth.User
-  alias Ueberauth.Strategy.Helpers
-
-  def callback(%{assigns: %{ueberauth_failure: _fails}} = conn, _params) do
+  alias Auth.Person
+
+  # https://github.com/dwyl/auth/issues/46
+  def admin(conn, _params) do
     conn
-    |> IO.inspect()
-    |> put_flash(:info, "Successfully authenticated.")
-    |> redirect(to: "/")
+    |> render(:welcome)
   end
```

Comment:
I don't think a pipe is necessary when only one function is called, we could have just:

```elixir
  def admin(conn, _params) do
    render(conn, :welcome)
  end
```

Reply:
@SimonLab agreed. The pipes are there because I was doing more things before. But we can/should definitely inline it. 👍